<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口小工具 2020.04</string>
  </property>
  <property name="windowIcon">
   <iconset resource="wzy.qrc">
    <normaloff>:/wzy/wzy.ico</normaloff>:/wzy/wzy.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="8,1,20">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>数据位</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="combox_name">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtBottom</enum>
              </property>
              <item>
               <property name="text">
                <string>COM1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM9</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>停止位</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="combox_stopbit"/>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="combox_parity"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>校验位</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="combox_baudrate"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="combox_databit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>控制流</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>串口名</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="combox_flow"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="push_serial">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>串口开关</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_serial">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_pause">
              <property name="text">
               <string>暂停接收</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_recover">
              <property name="text">
               <string>恢复接收</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="push_open_txet">
              <property name="text">
               <string>打开</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_save_txet">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_search_serial">
              <property name="text">
               <string>搜索串口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_debug">
              <property name="text">
               <string>初始配置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="push_send">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_input">
              <property name="text">
               <string/>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="push_clear">
              <property name="text">
               <string>清空</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_clear">
              <property name="text">
               <string>自动清空</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_about">
              <property name="text">
               <string>关于</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_exit">
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>325</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextBrowser" name="text_display">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="wzy.qrc"/>
 </resources>
 <connections/>
</ui>
